src/community/ogcapi/ogcapi-features/src/main/java/org/geoserver/ogcapi/features/FeatureService.java

```diff
@@ -324,8 +336,16 @@ public FeaturesResponse items(
         if (itemId != null) {
             filters.add(FF.id(FF.featureId(itemId)));
         }
+
         if (filter != null) {
-            Filter parsedFilter = filterParser.parse(filter, filterLanguage);
+            CoordinateReferenceSystem queryCRS = DefaultGeographicCRS.WGS84;
```

Comment:
I'd move this behavior in the FilterParser, there are other places where we can get a CRS as a string (e.g. STAC), better centralize it all.

Reply:
Moved to the `APIFilterParser`.